Core/TermsOfService.swift

```diff
@@ -40,7 +40,10 @@ public struct TermsOfService {
         if let classification = classification {
             return TermsOfService.classificationScores[classification]!
         }
-        return score < 0 ? -1 : 1
+
+        if score < 0 { return -1 }
```

Comment:
My only question before merge is: Is this nuanced enough to warrant a comment?

Reply:
Good point - I missed a unit test.  But yeah, I've added a comment to say we basically copied this from extension's JS